cmd/xl-v1-healing-common.go

```diff
@@ -184,7 +207,11 @@ func xlHealStat(xl xlObjects, partsMetadata []xlMetaV1, errs []error) HealObject
 	// Less than quorum erasure coded blocks of the object have the same create time.
 	// This object can't be healed with the information we have.
 	modTime, count := commonTime(listObjectModtimes(partsMetadata, errs))
-	if count < xl.readQuorum {
+
+	// get read quorum for this object
+	readQuorum, _, _ := objectQuorumFromMeta(xl, partsMetadata, errs)
```

Comment:
Not sure why are we ignoring error here..

Reply:
I missed this comment initially, but since `objectQuorumFromMeta` does the same test before throwing the error as done below this statement, IMO we can ignore the error here.

Reply:
We can clarify that here.

Reply:
sure will do

Reply:
actually you were right, we shouldn't ignore err here. Updated the PR